content/commands/hexpire/index.md

```diff
@@ -87,10 +87,8 @@ This means that all the operations that conceptually _alter_ the value stored at
 
 The TTL can be cleared using the [`HPERSIST`]({{< relref "/commands/hpersist" >}}) command, turning the hash key field back into a persistent field.
```

Comment:
I'm not sure if we should use "key field". I think that just "field" would be better

Reply:
Latest push.